server/common/overview/curator.ts

```diff
@@ -0,0 +1,119 @@
+import BigNumber from "bignumber.js";
+import { ParsedUrlQuery } from "querystring";
+import {Op, Sequelize, WhereOptions} from "sequelize";
+
+import models from "db/models";
+
+import { orderByProperty } from "helpers/array";
+import { caseInsensitiveEqual } from "helpers/db/conditionals";
+import { paginateArray } from "helpers/paginate";
+
+export default async function get(query: ParsedUrlQuery) {
+  const {
+    network,
+    chain,
+    address,
+    page = 1,
+    sortBy = "totalVotes",
+    order = "DESC"
+  } = query;
+
+  const curatorsWhere: WhereOptions = {
+    isCurrentlyCurator: true
+  };
+  const networkWhere: WhereOptions = {};
+  const chainWhere: WhereOptions = {};
+
+  if (address)
+    curatorsWhere.address = {
+      [Op.iLike]: `%${address.toString().toLowerCase()}%`
+    };
+
+  if (network)
+    networkWhere.name = caseInsensitiveEqual("network.name", network.toString().toLowerCase());
+
+  if (chain)
+    chainWhere.name = caseInsensitiveEqual("chainShortName", chain.toString().toLowerCase());
+
+  const curators = await models.curator.findAll({
```

Comment:
use should use the "same" query as we are doing on the leaderboard to avoid looping more than we need to